### DISCUSSION TITLE
Optionlly include 'Partner Shared' photos in 'Explore', 'Map', and 'Faces'.

### DISCUSSION THREAD

**Discussion by brayStorm** [immich-app/immich#12043] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Feels like the purpose of Partner Sharing. I want them meshed together everywhere.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3176
